src/trix/elements/trix_editor_element.js

```diff
@@ -453,6 +457,14 @@ export default class TrixEditorElement extends HTMLElement {
     this.editor?.loadHTML(this.defaultValue)
   }
 
+  // Element callbacks
+
+  attributeChangedCallback(name, oldValue, newValue) {
+    if (name === "initialized" && oldValue && oldValue !== newValue) {
```

Comment:
What happens to an editor with the `[initialized]` attribute that is disconnected from the page but written to Turbo's cache?

Would the `[initialized]` attribute need to be removed in the custom element's [disconnectedCallback()](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements#custom_element_lifecycle_callbacks)?

Reply:
I don't think this is necessary. Having that attribute in place won't prevent any initialization logic on re-connection. It's just to detect a modification where the trix element is modified/morphed without replacing it. I have tested the turbo cache scenario and everything works as expected.